src/tile/tile.scss

```diff
@@ -1,21 +1,18 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import '../style/index';
-@import '../icon/index';
 @import '../button/index';
+@import '../icon/index';
+@import '../surface/index';
 
 @mixin iui-tile {
   @include iui-reset;
+  @include iui-surface;
+  @include iui-focus($offset: $iui-xxs, $thickness: $iui-xxs);
```

Comment:
So non actionable tile also can be focused? I'd say only interactable ones should get it, no?

Reply:
I've moved the focus mixin within `.iui-actionable`.